packages/components/src/components/radio-group/radio-group.jsx

```diff
@@ -1,26 +1,53 @@
 import classNames from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
-import Radio from './radio.jsx';
 
-class RadioGroup extends React.PureComponent {
+class RadioGroup extends React.Component {
+    state = {
+        selectedOption: this.props.defaultOption,
+    };
+
+    // componentDidUpdate(prevProps) {
+    //     if (this.props.defaultChecked === prevProps.defaultChecked) {
+    //         return;
+    //     }
+
+    //     this.setState({ checked: this.props.defaultChecked });
+    // }
```

Comment:
Why is this commented out?

Reply:
I have to talk to Mahdi and ask about the use of it.